### PR TITLE
Add convenience script for running sonata simulations

### PR DESCRIPTION
**Pull request by GregAC**
We may not want this but I felt it would be helpful, at least at the hackathon, we can always remove it later.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ do you want to permanently mark this value as trusted (y/N)? y
 warning: ignoring untrusted substituter 'https://nix-cache.lowrisc.org/public/', you are not a trusted user.
 ```
 
-For Linux users, there is also an environment with the sonata simulator included:
+There is also an environment with the sonata simulator included:
```
Comment (GregAC) [inline README.md:58]: Not strictly true just yet but will be true shortly so may as well just change it here.

File: scripts/run_sim.sh [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+sonata_simulator -E $SONATA_SIM_BOOT_STUB -E $1
```
Comment (marnovandermaas) [inline scripts/run_sim.sh:3]: Where is `SONATA_SIM_BOOT_STUB` defined?

Comment (HU90m) [reply scripts/run_sim.sh:3]: In the nix environment: https://github.com/lowRISC/sonata-software/blob/d8ba1e1b74a6e3bf1ed92e4bd89e2c9d7f1550bb/flake.nix#L40

A check to ensure it is defined is a good idea.

Comment (marnovandermaas) [reply scripts/run_sim.sh:3]: Ah cool.

**Review by marnovandermaas — APPROVED** [review]
I haven't tested this but in principle happy to have this in the repository, at least temporarily.

File: scripts/run_sim.sh (hunk 2 above)
Comment (nbdd0121) [reply scripts/run_sim.sh:3]: It's in the nix devShell

**Pull request merged**